source/adminguide/ui.rst

```diff
@@ -273,8 +285,177 @@ Some assorted primary theme colours:
 - Green: #52C41A
 - Purple: #722ED1
 
-Advanced Customisation
-~~~~~~~~~~~~~~~~~~~~~~
+Contextual help documentation URLs can be customized with the help of `docBase` and `docHelpMappings` properties.
+To override a particular documentation URL, a mapping can be added for the URL path in the config. A documentation URL is formed by combining the `docBase` URL base and a path set in the source code. Adding a mapping for any particular path in the configuration will result in generating documetation URL with overridden path.
+By default, `docHelpMappings` lists all existing documentation URL suffixes, mapped to themselves, in the configuration file that are used in the code.
+
+.. parsed-literal::
+
+   {
+      ...
+      "docHelpMappings": {
```

Comment:
@shwstppr is it possible to maintain this mappings in separate file, and access from it?

Reply:
@sureshanaparti not sure if I completely understand you but in the actual config file for the newly created packages all mappings will be listed https://github.com/apache/cloudstack/pull/5014

Reply:
@shwstppr maybe you don't need to show all the mapping but the general idea of what they are and how users can change them - i.e. remap them by using the config.json file.

Reply:
@rhtyd @sureshanaparti can you please suggest what exactly should be done? Should I shorten the list here and move the complete list to a separate file with a link here? The remapping bit is already mentioned later in the doc. The complete list is only needed to give an idea to operators about what all docHelpSuffix(es) are used by the API, to make it easier to remap.

Reply:
Rewrite it simply as:
```
"docHelpMappings": {
 "map 1": "map1",
... snipped ...
 "map n": "map n"
 }
```
The intent is to tell users that the mappings will be in the config file that they need to re-map, we don't need to put the mapping list here - otherwise it will create a maintenance issue for docs (everytime the list changes, something needs to update it).

Reply:
@shwstppr If it is feasible to maintain these '_docHelpMappings_' in a separate json file, then you can go with it and let others know how these mappings can be updated for any new feature/improvement (may be, mention it in the README file). Otherwise, keep these mappings as it is and update the documentation about the mappings.